.github/workflows/nightly-builds.yml

```diff
@@ -134,6 +145,9 @@ jobs:
             -Dakka.test.timefactor=2 \
```

Comment:
I would like to add `-Xlog:gc` here, but that must be different parameters for jdk 8 and 11. What is the best way to define that when using a matrix like this?

Reply:
In the jdbc plugin tests we define some extraOpts in the matrix and then include it where needed. 

See https://github.com/akka/akka-persistence-jdbc/blob/master/.github/workflows/mysql-tests.yml